test/e2e/serial/tests/configuration.go

```diff
@@ -144,18 +144,18 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 			Expect(err).ToNot(HaveOccurred(), "cannot get %q in the cluster", nroKey.String())
 			initialNroOperObj := nroOperObj.DeepCopy()
 
-			workers, err := depnodes.GetWorkers(fxt.DEnv())
-			Expect(err).ToNot(HaveOccurred())
-			if len(workers) < 2 {
-				e2efixture.Skipf(fxt, "the test requires at least 2 worker nodes, found %d", len(workers))
-			}
-
-			targetIdx, ok := e2efixture.PickNodeIndex(workers)
-			Expect(ok).To(BeTrue())
-			targetedNode := workers[targetIdx]
+			nodesNameSet := e2enrt.AccumulateNames(nrts)
+			// nrts > 1 is checked in the setup
+			targetedNodeName, _ := nodesNameSet.PopAny()
+			targetedNode := &corev1.Node{}
```

Comment:
why a pointer? can be a local variable

Reply:
I presume it's because it passed to the client which accept a pointer.
it can be pass to the client using `&` but I saw it's a common usage to declare as a pointer.

Reply:
the golang escape analysis is probably doing a good job here, but I'd prefer in general to bias towards local variables.

Reply:
local var should be fine here as well, we'll just likely see more of the &<var> around